app/cafe.py

```diff
@@ -0,0 +1,24 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(
+            self,
+            visitor: dict
+    ) -> VaccineError | NotWearingMaskError | str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if date.today() > visitor["vaccine"]["expiration_date"]:
```

Comment:
You should consider using `.get()` methods to retrieve values from dicts in production code unless you are confident in the dict structure.